frontend/src/pages/ActivityReport/Pages/Review/HtmlReviewItem.js

```diff
@@ -19,7 +19,9 @@ const HtmlReviewItem = ({ label, name }) => {
   }
 
   values = values.map((v) => {
-    const defaultEditorState = getEditorState(v || '');
+    // eslint-disable-next-line no-param-reassign
+    if (v === '') v = 'None provided';
```

Comment:
If this is being used on a rich text editor the empty value might be a string with "<p></p>" might also want to check truthy.

Reply:
I'll look into updating this, thanks.

Reply:
Was this updated?

Reply:
Yes, it was updated.